server/routes/pinIPFS.ts

```diff
@@ -0,0 +1,19 @@
+import { Request, Response, NextFunction } from 'express';
+import { DB, sequelize } from '../database';
+import ipfs from '../util/ipfs';
+
+const pinIPFS = async (
+  models: DB,
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  try {
+    const ipfsHash = await ipfs(req.user.id, req.body);
```

Comment:
we should not just push the entirety of the req.body, we should make sure the client submits it inside a variable like `req.body.blob`, we need to make sure `blob` is not null, and then submit that `req.body.blob`.

Reply:
Fixed this, thanks